common/src/cellular_pktio.c

```diff
@@ -409,9 +409,9 @@ static _atRespType_t _getMsgType( const CellularContext_t * pContext,
     else
     {
         /* Check if prefix exist in pLine. */
-        atStatus = Cellular_ATIsPrefixPresent( pLine, &inputWithPrefix );
+        ( void ) Cellular_ATIsPrefixPresent( pLine, &inputWithPrefix );
```

Comment:
The initial value is false, so we can demonstrate the output variable directly instead of double demonstration of the return code and the output variable.